esp/esp/program/modules/forms/teacherreg.py

```diff
@@ -277,6 +289,12 @@ def clean(self):
         if class_size_optimal == '':
             cleaned_data['class_size_optimal'] = None
 
+        # If using grade ranges instead of min and max, extract min and max from grade range.
+        if cleaned_data.get('grade_range'):
+            grade_range = json.loads(cleaned_data.get('grade_range'))
```

Comment:
nit: clearer to do `grade_min, grade_max = json.loads(...)` or even `cleaned_data['grade_min'], cleaned_data['grade_max'] = json.loads(...)`